chiselFrontend/src/main/scala/chisel3/core/Bits.scala

```diff
@@ -1011,3 +1012,33 @@ object FixedPoint {
   }
 
 }
+
+/** Data type for Verilog inout */
+final class Analog private (width: Width) extends Element(width) {
```

Comment:
Sure, I'll add it to the ScalaDoc. It is the number of wires analogously to the width of a UInt